.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
You wanna move this into a separate PR so we can get  it in the repo a little quicker?

Reply:
Yes, but how would I do that ?

Reply:
1. copy this file
2. make a new branch based off main
3. commit and make pr

Reply:
Ok yea that was I was thinking